package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "renovate-config",
+  "version": "1.0.0",
+  "description": "Reelport's renovate configuration.",
+  "scripts": {
+    "validate": "ajv -s renovate-schema.json -d"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PicturePipe/renovate-config.git"
+  },
+  "author": "reelport GmbH <development@picturepipe.com>",
+  "private": true,
```

Comment:
Looking at the documentation of the `private` flag, it disables publication of the package.

I feel like there is no point in publishing this as a node package, so it seems to me like we should keep `private: true` set.

Is there a reason why we would want to publish this package or shouldn't guard against accidental publication?

Reply:
I always looked at the `private` option in conjunction with `license`.

Meaning: If an Open Source license is used, `private` should not be set to `false`.

But you are right: Although the code is licensed under an Open Source license, it's still a good idea to prevent it from being published on npm unless publication is intended.